### ISSUE TITLE
Dictionaries do not load under win7

### ISSUE THREAD

**Issue by J0hnnyCrash** [bartosz-antosik/vscode-spellright#115]
Hello!

I am trying to install SpellRight under Win7, followed the instructions, but no dictionaries are found:

1. created %APPDATA%\Code\Dictionaries\
2. Downloaded .dic and .aff files from https://github.com/titoBouzout/Dictionaries
3. verified that file format is UTF-8

On my Linux installation this worked pretty well, on win7 is does not... Am I missing something? VSCode and Spellright are on the latest Version.

Best Regards
  J0hnny

**Comment by bartosz-antosik** [comment]
Hm, would love to help, but somehow have no idea. I do not know what other information I would like to have, but one that comes to mind: Is your Windows/VSCode 32/64 bits?

Here it works in a analogous situation:

![image](https://user-images.githubusercontent.com/20492548/35982171-6c5aac06-0cef-11e8-971b-204df76e546b.png)

**Comment by bartosz-antosik** [comment]
I am going to close this issue. Shall you have some more data, e.g. if you can provide information requested above or any other clue that may be helpful to diagnose the situation, please post in this thread & I will revive it.

**Comment by dozius** [comment]
I was having this issue as well, so I looked at where the dictionaries are loaded. Looks like [this](https://github.com/bartosz-antosik/vscode-spellright/commit/6993ae782441bb9ebb9ecb76c11c0af1bafd2c5e) is the breaking change. 

storagePath [is not defined](https://github.com/Microsoft/vscode/issues/32353) when there is no workspace open. So, if you have no workspace open it's using the portable path for dictionaries, which obviously fails.

I tested this theory and sure enough, if I open a workspace the dictionaries are loaded. No workspace open, no dictionaries loaded.

Tested on win7, x64. Hope this helps!
